hw6/tasks/task1.py

```diff
@@ -0,0 +1,32 @@
+"""
+Задача 1: Правила ухаживания за собачками 🐕 +1
+
+Создайте функцию 'get_rules', которая возвращает правила по уходу за собачкой:
+'1. Регулярно расчёсывайте
+2. Гуляйте минимум 3 раза в день
+3. Часто играйте
+4. Не отказывайте в еде
+5. Наблюдайтесь у ветеринара'
+
+Под блоком if __name__ == '__main__':
+1. Выполните функцию.
+2. Распечатайте результат вызова функции.
+"""
+
+
+def get_rules():
+    # Загуглил как сделать многострочную строку, чтобы не писать пять раз print
+    # Выбрал этот вариант, который не делает лишних отступов из-за функции
+    rules = ("1. Регулярно расчёсывайте\n"
+            "2. Гуляйте минимум 3 раза в день\n"
+            "3. Часто играйте\n"
+            "4. Не отказывайте в еде\n"
+            "5. Наблюдайтесь у ветеринара\n"
+             )
+
+    return rules
+
+
+if __name__ == '__main__':
+    result = get_rules()
+    print(result)
```

Comment:
Здесь можно обойтись без временной переменной:
```suggestion
    print(get_rules())
```